test/js/node/http2/connectrpc-repro/test_connect.py

```
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
```

Ruff (0.14.13)

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)